ballerina/tests/from_json_string_test.bal

```diff
@@ -1552,7 +1552,7 @@ isolated function testParseStringNegative3() returns Error? {
 @test:Config
 isolated function testParseStringNegative4() returns Error? {
     string str = string `{
-        name: "John"
+        "name": "John"
```

Comment:
why we need this change?

Reply:
Json Object key should be present within `""`